docs/how-to/data-plane-installation.md

```diff
@@ -0,0 +1,72 @@
+# Installing Kuadrant data-plane into an existing OCM Managed Cluster
```

Comment:
We might change this to `Service Protection` ?

Reply:
better, yes